lib/src/model.dart

```diff
@@ -1258,7 +1258,17 @@ class Parameter extends ModelElement {
 
   @override
   String get _href {
-    throw new UnsupportedError('parameters do not have hrefs');
+    var p = _parameter.enclosingElement;
```

Comment:
Also used to resolve references in comments - parameters are linked in comments for methods/functions.